File: vnext/Microsoft.ReactNative.Managed/ReactPackageProvider.cs
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

namespace Microsoft.ReactNative.Managed
{
    /// <summary>
    /// This provider needs to remain for projects that still reference this code via a source package instead of a library.
    /// </summary>
    public class ReactPackageProvider : IReactPackageProvider
    {
        public void CreatePackage(IReactPackageBuilder packageBuilder)
        {
            packageBuilder.AddReflectionReactPackageProvider<ReactPackageProvider>();
        }
    }
}

File: packages/node-rnw-rpc/windows/NodeRpc/ReactPackageProvider.cpp
#include "pch.h"
#include "ReactPackageProvider.h"
#if __has_include("ReactPackageProvider.g.cpp")
#include "ReactPackageProvider.g.cpp"
#endif

using namespace winrt::Microsoft::ReactNative;

namespace winrt::NodeRpc::implementation {

void ReactPackageProvider::CreatePackage(IReactPackageBuilder const & /*packageBuilder*/) noexcept {
  // Noop
}

} // namespace winrt::NodeRpc::implementation
